Cinebase/Core/Formatters/DateFormatter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DateFormatter.swift
+//  Cinebase
+//
+//  Created by Gustavo Villar on 5/16/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import Foundation
+
+class DateFormatter {
+    
+    enum Style {
+        case date
+        case dateTime
+    }
+    
+    private let formatter = Foundation.DateFormatter()
+    
+    func format(_ date: Date, style: Style) -> String {
+        switch style {
```

Comment:
is there a way to put the values of each style as individual vars, it will be cleaner to modify?